.gitignore

```diff
@@ -17,4 +17,8 @@ target/
 # Branch switching
 generated/
 .DS_Store
```

Comment:
I'm sorry, the `.DS_Store` file was already declared. I don't think we have to declare full paths.